archive/settings.py

```diff
@@ -234,6 +239,18 @@
 PROCESSED_EXCHANGE_ENABLED = ast.literal_eval(os.getenv('PROCESSED_EXCHANGE_ENABLED', 'True'))
 PROCESSED_EXCHANGE_NAME = os.getenv('PROCESSED_EXCHANGE_NAME', 'archived_fits')
 
+# Additional Customization
+ZIP_DOWNLOAD_FILENAME_BASE = os.getenv('ZIP_DOWNLOAD_FILENAME_BASE', 'ocs_archive_data')
+ZIP_DOWNLOAD_MAX_UNCOMPRESSED_FILES = os.getenv('ZIP_DOWNLOAD_MAX_UNCOMPRESSED_FILES', 10)
+NAVBAR_TITLE_TEXT = os.getenv('NAVBAR_TITLE_TEXT', 'Science Archive API')
+NAVBAR_TITLE_URL = os.getenv('NAVBAR_TITLE_URL', 'https://archive.lcogt.net')
+TERMS_OF_SERVICE_URL = os.getenv('TERMS_OF_SERVICE_URL', 'https://lco.global/policies/terms/')
+
+# Pagination limits, ensuring the value is an integer
+try: 
+    PAGINATION_DEFAULT_LIMIT = int(os.getenv('PAGINATION_DEFAULT_LIMIT'))
+except (ValueError, TypeError):  # handle non-integer or nonexistant value in the int() method
+    PAGINATION_DEFAULT_LIMIT = 100
```

Comment:
The value needs to be an int (as used in `archive/frames/pagination`). Current strategy is to try to convert the env value to an int, and apply the default if it doesn't work. But this might result in a silent error. Is there a better way to do this?

Reply:
IMO I'd just do `PAGINATION_DEFAULT_LIMIT = int(os.getenv('PAGINATION_DEFAULT_LIMIT', 100))`

Hopefully if someone mis-configures this it'll be obvious and just crash on startup with the error.

Reply:
Oh right, I guess the crash in that scenario is what we want. Thanks for the suggestion!